src/position/position.cairo

```diff
@@ -59,3 +59,38 @@ impl DefaultPosition of Default<Position> {
         }
     }
 }
+/// Trait for Position.
+#[generate_trait]
+impl PositionImpl of PositionTrait {
```

Comment:
Create trait and implementation of getters and setters

Reply:
Is this needed since we can directly access the fields of the struct ?

Reply:
I saw that these getters and setters are used quite a few times in the gmx repository so I thought to create this implementation so it will be convenient for everyone

Reply:
In GMX they're using them because they have nested structs. For example they need to do `props.addresses.account`. But we don't have that, we can directly do props.account, so I think we don't need it

Reply:
Gotcha thanks for the context, will remove this 🤙